🌐 [i18n-KO] Translated `conceptual/philosophy.md` and 3 other documents to Korean

mreraser opened the pull request:
# What does this PR do?

Hello!😀

Translated 
`conceptual/contribution.md`
`conceptual/ethical_guidelines.md`
`conceptual/evaluation.md`
`conceptual/philosophy.md`
file of the documentation to Korean.

Started from [#7940](https://github.com/huggingface/diffusers/issues/7940)

Thank you so much for your valuable review. 

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Check Inline TOC (e.g. [[lowercased-header]])
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)
I have great respect for your previous contributions, and I would really appreciate if @wonhyeongseo and the PseudoLab group could review this.

@jungnerd, @heuristicwave, @harheem, @cjfghk5697, @seoulsky-field, @nuatmochoi, @boyunJang, @SeungAhSon, @seoyoung-3060, @junejae


## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?


## Who can review? (Final)

Hello @stevhliu and @sayakpaul!
Thank you for all your hard work and efforts. I would really appreciate if you could review this.

<!-- Your PR will be replied to more quickly if you can figure out the right person to tag with @.

 If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
 Please tag fewer than 3 people.

Core library:

- Schedulers: @yiyixuxu 
- Pipelines:  @sayakpaul @yiyixuxu @DN6
- Training examples: @sayakpaul 
- Docs: @stevhliu and @sayakpaul
- JAX and MPS: @pcuenca
- Audio: @sanchit-gandhi
- General functionalities: @sayakpaul @yiyixuxu @DN6

Integrations:

- deepspeed: HF Trainer/Accelerate: @pacman100

HF projects:

- accelerate: [different repo](https://github.com/huggingface/accelerate)
- datasets: [different repo](https://github.com/huggingface/datasets)
- transformers: [different repo](https://github.com/huggingface/transformers)
- safetensors: [different repo](https://github.com/huggingface/safetensors)

-->

stevhliu commented on docs/source/ko/conceptual/contribution.md:
```diff
@@ -0,0 +1,512 @@
+
+`main` 브랜치 위에서 **절대** 작업하지 마세요.
+
+1. 가상 환경에서 다음 명령을 실행하여 개발 환경을 설정하세요:
```
```suggestion
4. 가상 환경에서 다음 명령을 실행하여 개발 환경을 설정하세요:
```

stevhliu reviewed (commented):
Great job in translating such long and dense docs! The structure looks good to me and once PseudoLab has reviewed the actual translation, we can merge 🤗

cjfghk5697 commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+  - [**안전 검사기**](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/safety_checker.py): 이미지가 생성된 후에 이미자가 임베딩 공간에서 일련의 하드코딩된 유해 개념의 클래스일 확률을 확인하고 비교합니다. 유해 개념은 역공학을 방지하기 위해 의도적으로 숨겨져 있습니다.
+
+- **Hub에서의 단계적인 배포**: 특히 민감한 상황에서는 일부 저장소에 대한 액세스를 제한해야 합니다. 이 단계적인 배포는 저장소의 작성자가 사용에 대해 더 많은 통제를 가질 수 있도록 중간 단계로 작용합니다.
```
```suggestion
- **Hub에서의 단계적인 배포**: 특히 민감한 상황에서는 일부 리포지토리에 대한 액세스를 제한해야 합니다. 이 단계적인 배포는 리포지토리의 작성자가 사용에 대해 더 많은 통제를 가질 수 있도록 중간 단계로 작용합니다.
```

cjfghk5697 commented on docs/source/ko/conceptual/philosopy.md:
```diff
@@ -0,0 +1,103 @@
+## 쉬움보다는 간단함을
+
+PyTorch에서는 **명시적인 것이 암시적인 것보다 낫다**와 **단순한 것이 복잡한 것보다 낫다**라고 말합니다. 이 설계 철학은 라이브러리의 여러 부분에 반영되어 있습니다:
+- [`DiffusionPipeline.to`](https://huggingface.co/docs/diffusers/main/en/api/diffusion_pipeline#diffusers.DiffusionPipeline.to)와 같은 메서드를 사용하여 사용자가 장치 관리를 할 수 있도록 PyTorch의 API를 따릅니다.
```
```suggestion
- [`DiffusionPipeline.to`](https://huggingface.co/docs/diffusers/main/en/api/diffusion_pipeline#diffusers.DiffusionPipeline.to)와 같은 메소드를 사용하여 사용자가 장치 관리를 할 수 있도록 PyTorch의 API를 따릅니다.
```

cjfghk5697 commented on docs/source/ko/conceptual/philosopy.md:
```diff
@@ -0,0 +1,103 @@
+- 파이프라인은 매우 가독성이 좋고, 이해하기 쉽고, 쉽게 조정할 수 있도록 설계되어야 합니다.
+- 파이프라인은 서로 상호작용하고, 상위 수준 API에 쉽게 통합할 수 있도록 설계되어야 합니다.
+- 파이프라인은 사용자 인터페이스가 feature-complete하지 않게 하는 것을 목표로 합니다. future-complete한 사용자 인터페이스를 원한다면 [InvokeAI](https://github.com/invoke-ai/InvokeAI), [Diffuzers](https://github.com/abhishekkrthakur/diffuzers), [lama-cleaner](https://github.com/Sanster/lama-cleaner)를 참조해야 합니다.
+- 모든 파이프라인은 오로지 `__call__` 메서드를 통해 실행할 수 있어야 합니다. `__call__` 인자의 이름은 모든 파이프라인에서 공유되어야 합니다.
```
```suggestion
- 모든 파이프라인은 오로지 `__call__` 메소드를 통해 실행할 수 있어야 합니다. `__call__` 인자의 이름은 모든 파이프라인에서 공유되어야 합니다.
```

cjfghk5697 commented on docs/source/ko/conceptual/philosopy.md:
```diff
@@ -0,0 +1,103 @@
+- 하나의 스케줄러 Python 파일은 하나의 스케줄러 알고리즘(논문에서 정의된 것과 같은)에 해당합니다.
+- 스케줄러가 유사한 기능을 공유하는 경우, `#Copied from` 메커니즘을 사용할 수 있습니다.
+- 모든 스케줄러는 `SchedulerMixin`과 `ConfigMixin`을 상속합니다.
+- [`ConfigMixin.from_config`](https://huggingface.co/docs/diffusers/main/en/api/configuration#diffusers.ConfigMixin.from_config) 메서드를 사용하여 스케줄러를 쉽게 교체할 수 있습니다. 자세한 내용은 [여기](../using-diffusers/schedulers.md)에서 설명합니다.
```
```suggestion
- [`ConfigMixin.from_config`](https://huggingface.co/docs/diffusers/main/en/api/configuration#diffusers.ConfigMixin.from_config) 메소드를 사용하여 스케줄러를 쉽게 교체할 수 있습니다. 자세한 내용은 [여기](../using-diffusers/schedulers.md)에서 설명합니다.
```

cjfghk5697 reviewed (commented):
HuggingFace Python Library에 참고하여 일단 메서드 -> 메소드, 저장소 ->리포지토리로 변경했습니다!

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+specific language governing permissions and limitations under the License.
+-->
+
+# 🧨 Diffusers의 윤리 지침
```
```suggestion
# 🧨 Diffusers의 윤리 지침[[-diffusers-ethical-guidelines]]
```

mreraser replied:
안녕하십니까! @heuristicwave님. 귀중한 시간 내주셔서 리뷰 해주시는 것에 진심으로 감사드립니다 😄

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+# 🧨 Diffusers의 윤리 지침
+
+## 서문
```
```suggestion
## 서문[[preamble]]
```

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+우리는 위험을 지속적으로 추적하고 커뮤니티의 응답과 소중한 피드백에 따라 다음 지침을 조정할 것입니다.
+
+
+## 범위
```
```suggestion
## 범위[[scope]]
```

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+Diffusers 커뮤니티는 프로젝트의 개발에 다음과 같은 윤리 지침을 적용하며, 특히 윤리적 문제와 관련된 민감한 주제에 대한 커뮤니티의 기여를 조정하는 데 도움을 줄 것입니다.
+
+
+## 윤리 지침
```
```suggestion
## 윤리 지침[[ethical-guidelines]]
```

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+- **책임**: 우리는 팀워크를 통한 커뮤니티로서 이 기술의 잠재적인 위험과 위험을 예측하고 완화하는 데 대한 공동 책임을 가지고 있습니다.
+
+
+## 구현 사례: 안전 기능과 메커니즘
```
```suggestion
## 구현 사례: 안전 기능과 메커니즘[[examples-of-implementations-safety-features-and-mechanisms]]
```

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+이 기술의 실제 적용과 사회에 미칠 수 있는 부정적인 영향을 고려하여 Diffusers 라이브러리의 개발, 사용자 기여 및 사용에 윤리 지침을 제공하는 것이 중요하다고 생각합니다.
+
+이 기술을 사용하는 데 연관된 위험은 아직 조사 중이지만, 몇 가지 예를 들면: 예술가들에 대한 저작권 문제; 딥 페이크의 악용; 부적절한 맥락에서의 성적 콘텐츠 생성; 동의 없는 impersonation; 사회적인 편견으로 인해 억압되는 그룹들에 대한 해로운 영향입니다.
```
```suggestion
이 기술을 사용함에 따른 위험은 여전히 검토 중이지만, 몇 가지 예를 들면: 예술가들에 대한 저작권 문제; 딥 페이크의 악용; 부적절한 맥락에서의 성적 콘텐츠 생성; 동의 없는 사칭; 소수자 집단의 억압을 영속화하는 유해한 사회적 편견 등이 있습니다.
```

heuristicwave replied:
'impersonation'을 '사칭'으로 번역 및 조금 더 매끄럽게 변경해 보았습니다.

mreraser replied:
아 그렇군요! ☺️ 적당한 단어를 못 찾아서 그대로 두었는데 좋은 번역인 것 같습니다. 감사합니다!

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+## 윤리 지침
+
+다음 윤리 지침은 일반적으로 적용되지만, 기술적 선택을 할 때 윤리적으로 민감한 문제를 다룰 때 주로 적용할 것입니다. 또한, 해당 기술의 최신 동향과 관련된 신규 위험에 따라 시간이 지남에 따라 이러한 윤리 원칙을 조정할 것을 약속합니다.
```
```suggestion
다음 윤리 지침은 일반적으로 적용되지만, 민감한 윤리적 문제와 관련하여 기술적 선택을 할 때 이를 우선적으로 적용할 것입니다. 나아가, 해당 기술의 최신 동향과 관련된 새로운 위험이 발생함에 따라 이러한 윤리 원칙을 조정할 것을 약속드립니다.
```

heuristicwave replied:
동어 반복을 피하고 문장을 조금 더 매끄럽게 변경해보았습니다.

jungnerd reviewed (commented):
`philosophy.md` 일부 리뷰했으니 확인 부탁드립니다!

mreraser commented on docs/source/ko/conceptual/philosopy.md:
```diff
@@ -0,0 +1,103 @@
+specific language governing permissions and limitations under the License.
+-->
+
+# 철학
```
안녕하십니까 @jungnerd 님! 귀중한 시간 내셔서 리뷰해주심에 매우 감사합니다 😄

mreraser commented on docs/source/ko/conceptual/philosopy.md:
```diff
@@ -0,0 +1,103 @@
+## 성능보다는 사용성을
+
+- Diffusers는 많은 내장 성능 향상 기능을 갖고 있지만 (자세한 내용은 [메모리와 속도](https://huggingface.co/docs/diffusers/optimization/fp16) 참조), 모델은 항상 가장 높은 정밀도와 최소한의 최적화로 로드됩니다. 따라서 기본적인 diffusion 파이프라인은 따로 정의하지 않는다면 CPU에서 float32 정밀도로 인스턴스화됩니다. 이는 다양한 플랫폼과 가속기에서의 사용성을 보장하며, 라이브러리를 실행하기 위해 복잡한 설치가 필요하지 않음을 의미합니다.
+- Diffusers는 **가벼운** 패키지를 지향하며, 따라서 매우 적은 필수 종속성이 있고, 대신에 성능을 향상시킬 수 있는 많은 선택적 종속성이 있습니다 (`accelerate`, `safetensors`, `onnx` 등). 우리는 라이브러리를 가능한 한 가볍게 유지하여 다른 패키지와의 종속성으로 인해 걱정할 필요가 없도록 노력합니다.
```
`지향하기 때문에V필수 종속성은 거의 없지만` : '때문에'와 '필수' 사이에 띄어쓰기가 되어야 할 것 같습니다!

mreraser commented:
> `philosophy.md` 일부 리뷰했으니 확인 부탁드립니다!

넵 확인하였습니다! 감사합니다 ☺️

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+다음 윤리 지침은 일반적으로 적용되지만, 기술적 선택을 할 때 윤리적으로 민감한 문제를 다룰 때 주로 적용할 것입니다. 또한, 해당 기술의 최신 동향과 관련된 신규 위험에 따라 시간이 지남에 따라 이러한 윤리 원칙을 조정할 것을 약속합니다.
+
+- **투명성**: PR 관리, 사용자에게 선택 사항을 설명하고 기술적 결정을 내리는 데 있어서 투명성을 유지하는 것에 약속합니다.
```
```suggestion
- **투명성**: 우리는 PR을 관리하고, 사용자에게 우리의 선택을 설명하며, 기술적 의사결정을 내릴 때 투명성을 유지할 것을 약속합니다.
```

heuristicwave replied:
가이드라인을 설명하는 만큼, 조금 더 친절한 표현으로 변경해 보았습니다.

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+- **투명성**: PR 관리, 사용자에게 선택 사항을 설명하고 기술적 결정을 내리는 데 있어서 투명성을 유지하는 것에 약속합니다.
+
+- **일관성**: 프로젝트 관리에서 사용자에게 동일한 수준의 주의를 보장하고 기술적으로 안정하고 일관성 있게 유지하는 것에 약속합니다.
```
```suggestion
- **일관성**: 우리는 프로젝트 관리에서 사용자들에게 동일한 수준의 관심을 보장하고 기술적으로 안정되고 일관된 상태를 유지할 것을 약속합니다.
```

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+- **일관성**: 프로젝트 관리에서 사용자에게 동일한 수준의 주의를 보장하고 기술적으로 안정하고 일관성 있게 유지하는 것에 약속합니다.
+
+- **간결성**: Diffusers 라이브러리를 사용하고 활용하기 쉽게 만들기 위해 프로젝트의 목표를 간결하고 일관성 있게 유지하는 것에 약속합니다.
```
```suggestion
- **간결성**: Diffusers 라이브러리를 사용하고 활용하기 쉽게 만들기 위해, 프로젝트의 목표를 간결하고 일관성 있게 유지할 것을 약속합니다.
```

heuristicwave replied:
이전 제안과 비슷한 뉘앙스로 살짝 바꿔보았습니다.

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+- **간결성**: Diffusers 라이브러리를 사용하고 활용하기 쉽게 만들기 위해 프로젝트의 목표를 간결하고 일관성 있게 유지하는 것에 약속합니다.
+
+- **접근성**: Diffusers 프로젝트는 기술적 전문 지식 없이도 실행에 도움을 줄 수 있는 기여자들이 참여할 수 있도록 진입장벽을 낮추는 데 도움을 줍니다. 이렇게 하는 것은 연구 자료를 커뮤니티에게 더욱 접근 가능하게 만듭니다.
```
```suggestion
- **접근성**: Diffusers 프로젝트는 기술적 전문 지식 없어도 프로젝트 운영에 참여할 수 있는 기여자의 진입장벽을 낮춥니다. 이를 통해 연구 결과물이 커뮤니티에 더 잘 접근할 수 있게 됩니다. 
```

heuristicwave replied:
'can help run it' 을 Diffusers '프로젝트 운영에 참여할 수 있는' 으로 바꾸어 보았습니다.

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+- **접근성**: Diffusers 프로젝트는 기술적 전문 지식 없이도 실행에 도움을 줄 수 있는 기여자들이 참여할 수 있도록 진입장벽을 낮추는 데 도움을 줍니다. 이렇게 하는 것은 연구 자료를 커뮤니티에게 더욱 접근 가능하게 만듭니다.
+
+- **재현성**: Diffusers 라이브러리를 통해 제공되는 상류 코드, 모델 및 데이터셋의 재현성에 대해 투명하게 공개할 것을 목표로 합니다.
```
```suggestion
- **재현성**: 우리는 Diffusers 라이브러리를 통해 제공되는 업스트림(upstream) 코드, 모델 및 데이터셋의 재현성에 대해 투명하게 공개할 것을 목표로 합니다.
```

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+- **재현성**: Diffusers 라이브러리를 통해 제공되는 상류 코드, 모델 및 데이터셋의 재현성에 대해 투명하게 공개할 것을 목표로 합니다.
+
+- **책임**: 우리는 팀워크를 통한 커뮤니티로서 이 기술의 잠재적인 위험과 위험을 예측하고 완화하는 데 대한 공동 책임을 가지고 있습니다.
```
```suggestion
- **책임**: 우리는 커뮤니티와 팀워크를 통해, 이 기술의 잠재적인 위험과 위험을 예측하고 완화하는 데 대한 공동 책임을 가지고 있습니다.
```

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+## 구현 사례: 안전 기능과 메커니즘
+
+팀은 diffusion 기술과 관련된 잠재적인 윤리적 및 사회적 위험에 대처하기 위한 기술적 및 비기술적 도구를 제공하고자 하고 있습니다. 또한, 커뮤니티의 참여는 이러한 기능의 구현과 함께 우리와 함께 인식을 높이는 데 매우 중요합니다.
```
```suggestion
팀은 diffusion 기술과 관련된 잠재적인 윤리 및 사회적 위험에 대처하기 위한 기술적 및 비기술적 도구를 제공하고자 하고 있습니다. 또한, 커뮤니티의 참여는 이러한 기능의 구현하고 우리와 함께 인식을 높이는 데 매우 중요합니다.
```

heuristicwave replied:
'적', '함께' 동어 반복을 다른 방법으로 표현해 보았습니다.

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+- [**커뮤니티 탭**](https://huggingface.co/docs/hub/repositories-pull-requests-discussions): 이를 통해 커뮤니티는 프로젝트에 대해 토론하고 더 나은 협력을 할 수 있습니다.
+
+- **편향 탐색 및 평가**: Hugging Face 팀은 Stable Diffusion 모델의 편향을 대화식으로 시각화하는 [공간](https://huggingface.co/spaces/society-ethics/DiffusionBiasExplorer)을 제공합니다. 이런 의미에서, 우리는 편향 탐색 및 평가를 지원하고 장려합니다.
```
```suggestion
- **편향 탐색 및 평가**: Hugging Face 팀은 Stable Diffusion 모델의 편향성을 대화형으로 보여주는 [space](https://huggingface.co/spaces/society-ethics/DiffusionBiasExplorer)을 제공합니다. 이런 의미에서, 우리는 편향 탐색 및 평가를 지원하고 장려합니다.
```

heuristicwave replied:
space 대명사는 그래도 유지하며, 표현을 약간 변경해 보았습니다.

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+  - [**안전 검사기**](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/safety_checker.py): 이미지가 생성된 후에 이미자가 임베딩 공간에서 일련의 하드코딩된 유해 개념의 클래스일 확률을 확인하고 비교합니다. 유해 개념은 역공학을 방지하기 위해 의도적으로 숨겨져 있습니다.
+
+- **Hub에서의 단계적인 배포**: 특히 민감한 상황에서는 일부 저장소에 대한 액세스를 제한해야 합니다. 이 단계적인 배포는 저장소의 작성자가 사용에 대해 더 많은 통제를 가질 수 있도록 중간 단계로 작용합니다.
```
```suggestion
- **Hub에서의 단계적인 배포**: 특히 민감한 상황에서는 일부 리포지토리에 대한 접근을 제한해야 합니다. 이 단계적인 배포는 중간 단계로, 리포지토리 작성자가 사용에 대한 더 많은 통제력을 갖게 합니다. 
```

heuristicwave replied:
TTA 사전을 참고하여, [access](https://terms.tta.or.kr/dictionary/dictionaryView.do?word_seq=086256-1) 를 접근으로 및 표현을 약간 변경해 보았습니다.

heuristicwave commented on docs/source/ko/conceptual/ethical_guidelines.md:
```diff
@@ -0,0 +1,63 @@
+
+- **Hub에서의 단계적인 배포**: 특히 민감한 상황에서는 일부 저장소에 대한 액세스를 제한해야 합니다. 이 단계적인 배포는 저장소의 작성자가 사용에 대해 더 많은 통제를 가질 수 있도록 중간 단계로 작용합니다.
+
+- **라이센싱**: [OpenRAILs](https://huggingface.co/blog/open_rail)는 더 책임 있는 사용을 보장하는 일련의 제한을 가지면서도 무료 액세스를 보장할 수 있도록 하는 새로운 유형의 라이센스입니다.
```
```suggestion
- **라이선싱**: [OpenRAILs](https://huggingface.co/blog/open_rail)와 같은 새로운 유형의 라이선싱을 통해 자유로운 접근을 보장하면서도 더 책임 있는 사용을 위한 일련의 제한을 둘 수 있습니다.
```

heuristicwave replied:
외래어표기법 '라이선스'를 차용해 고쳐 보았습니다.

여기까지 윤리 지침 리뷰를 마무리 하겠습니다! 감사합니다!

mreraser replied:
@heuristicwave 감사합니다! 

다시 한번 많은 시간 내주셔서 번역 리뷰해주심에 감사합니다! 좋은 하루 되세요😊

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/diffusers/pr_8294). All of your documentation changes will be reflected on that endpoint. The docs are available until 30 days after the last update.

cjfghk5697 commented on docs/source/ko/conceptual/evaluation.md:
```diff
@@ -0,0 +1,554 @@
+specific language governing permissions and limitations under the License.
+-->
+
+# Diffusion 모델 평가하기
```
```suggestion
# Diffusion 모델 평가하기[[evaluating-diffusion-models]]
```

cjfghk5697 commented on docs/source/ko/conceptual/evaluation.md:
```diff
@@ -0,0 +1,554 @@
+
+이 문서에서 보여진 방법들은 기반 생성 모델을 고정시키고 다양한 [노이즈 스케줄러](https://huggingface.co/docs/diffusers/main/en/api/schedulers/overview)를 평가하는 데에도 사용할 수 있습니다.
+
+## 시나리오
```
```suggestion
## 시나리오[[scenarios]]
```

cjfghk5697 commented on docs/source/ko/conceptual/evaluation.md:
```diff
@@ -0,0 +1,554 @@
+- 입력 이미지에 추가로 조건을 건 텍스트로 안내된 이미지 생성 (예: [`StableDiffusionImg2ImgPipeline`](https://huggingface.co/docs/diffusers/main/en/api/pipelines/stable_diffusion/img2img) 및 [`StableDiffusionInstructPix2PixPipeline`](https://huggingface.co/docs/diffusers/main/en/api/pipelines/pix2pix)).
+- 클래스 조건화된 이미지 생성 모델 (예: [`DiTPipeline`](https://huggingface.co/docs/diffusers/main/en/api/pipelines/dit)).
+
+## 정성적 평가
```
```suggestion
## 정성적 평가[[qualitative-evaluation]]
```

cjfghk5697 commented on docs/source/ko/conceptual/evaluation.md:
```diff
@@ -0,0 +1,554 @@
+
+</Tip>
+
+## 정량적 평가
```
```suggestion
## 정량적 평가[[quantitative-evaluation]]
```

cjfghk5697 commented on docs/source/ko/conceptual/evaluation.md:
```diff
@@ -0,0 +1,554 @@
+- CLIP 방향성 유사도
+- FID
+
+### 텍스트 안내 이미지 생성
```
```suggestion
### 텍스트 안내 이미지 생성[[text-guided-image-generation]]
```

cjfghk5697 commented on docs/source/ko/conceptual/evaluation.md:
```diff
@@ -0,0 +1,554 @@
+
+</Tip>
+
+### 이미지 조건화된 텍스트-이미지 생성
```
```suggestion
### 이미지 조건화된 텍스트-이미지 생성[[image-conditioned-text-to-image-generation]]
```

cjfghk5697 commented on docs/source/ko/conceptual/evaluation.md:
```diff
@@ -0,0 +1,554 @@
+
+***위의 메트릭을 사용하면 클래스 조건이 있는 모델을 평가할 수 있습니다. 예를 들어, [DiT](https://huggingface.co/docs/diffusers/main/en/api/pipelines/dit). 이는 ImageNet-1k 클래스에 조건을 걸고 사전 훈련되었습니다.***
+
+### 클래스 조건화 이미지 생성
```
```suggestion
### 클래스 조건화 이미지 생성[[class-conditioned-image-generation]]
```

stevhliu commented:
Feel free to let me know when you've addressed all the feedback and are ready to merge! 🤗

mreraser commented:
> Feel free to let me know when you've addressed all the feedback and are ready to merge! 🤗

Thank you! I believe I've addressed all the feedback and am ready for the merge. I appreciate all your hard work :)

stevhliu reviewed (approved):
Thanks again for the contribution! 🤗
